pkg/state/state_test.go

```diff
@@ -2789,6 +2799,16 @@ func TestPrepareSyncReleases(t *testing.T) {
 				ReuseValues: true,
 			},
 		},
+		{
+			name:  "force-reset-values",
+			flags: []string{"--reset-values"},
+			syncOptions: &SyncOpts{
+				ResetValues: true,
+			},
+			helmDefaults: &HelmSpec{
+				ReuseValues: true,
+			},
+		},
```

Comment:
add a test case for --reset-values and --reuse-values in the same.

Reply:
added